dev-lang/cppfront/cppfront-0.7.4.recipe

```diff
@@ -36,6 +36,8 @@ BUILD()
 
 INSTALL()
 {
-	mkdir -p $prefix/bin
-	cp cppfront $prefix/bin
+	mkdir -p $binDir
+	cp cppfront $binDir
```

Comment:
This was correct before. `$binDir` includes the secondaryArch subdir on secondary architectures (i.e. x86), which is not wanted here.